scripts/verify-template.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const puppeteer = require('puppeteer');
+
+/**
+ * This script is used to check for runtime errors in the starter templates
+ * create-sumup-next-app and create-sumup-react-app.
+ */
+(async () => {
+  const expected = 'Welcome to SumUp';
+
+  try {
+    const browser = await puppeteer.launch();
+    const page = await browser.newPage();
+
+    await page.goto('http://localhost:3000');
+    await page.screenshot({ path: 'example.png' });
```

Comment:
Is this saving the screenshot as a build artifact? Don't we need more stuff in the GH action for this?

Reply:
Whoops, that is a leftover from the example that I copied.

Reply:
Removed in 32095f1.